ui/desktop/README.md

```diff
@@ -83,18 +83,12 @@ To run as a desktop app:
 
 - `yarn start:desktop`
 
-The Boundary CLI is downloaded and extracted to `electron-app/cli/` folder as part of
-build. CLI version is defined in `electron-app/config/cli.js`.
+The Boundary CLI is NOT downloaded by default, to download and extract the CLI to `electron-app/cli` folder as part of the build, you need to set environment variable `CLI_SETUP` to true. Example: `CLI_SETUP=true yarn start:desktop`.
+CLI version is defined in `electron-app/config.cli.js`.
```

Comment:
```suggestion
The CLI version is defined in `electron-app/config/cli/VERSION`.
```
Don't we define it in the `VERSION` file?

Reply:
You are correct, I made the confusion here on wording. When I mean the version is defined within config.cli.js I mean the script will generate what version to download via the url, but you are totally correct, the CLI version is defined within the VERSION file. Accepting your changes 😉 

Thanks for the catch!